### ISSUE TITLE
2018 general election precinct return problems

### ISSUE THREAD

**Issue by agadjanian** [openelections/openelections-data-mo#12]
There seems to be an issue with the raw 2018 Missouri precinct data. For the statewide ballot items--constitutional amendments and propositions--the no votes appear to be missing (some votes are labelled "yes" and some don't have a candidate label, and all of these add up to basically the total number of yes votes reported in the official results).

**Comment by dwillis** [comment]
@agadjanian I'll check on it, thanks. Our focus is on a subset of contests (we don't include ballot items for most states), so it might take a bit.

**Comment by agadjanian** [comment]
Ah okay, thanks for clarifying. That makes sense, though let me know if you find anything when/if you get to it.

**Comment by lephead** [comment]
Yes I made my chart and some counties did not include lines for candidates who received no votes and there is one St Louis City precinct labeled St Louis County.
